protostar/cheatable_starknet/callable_hint_locals/call_hint_local.py

```diff
@@ -29,19 +30,21 @@ def name(self) -> str:
 
     def _build(
         self,
-    ) -> Callable[[RawAddress, str, Optional[CairoOrPythonData]], CairoOrPythonData]:
+    ) -> Callable[
+        [RawAddress, CairoShortString, Optional[CairoOrPythonData]], CallResult
```

Comment:
```suggestion
        [RawAddress, CairoShortString, Optional[CairoData]], CallResult
```

Reply:
Ok, but I'm going to leave this as a TODO https://github.com/software-mansion/protostar/issues/1754

Reply:
[597df95](https://github.com/software-mansion/protostar/pull/1726/commits/597df95a518c34c63c1eeef0bb2009dcba09cd74)